dev/tasks/homebrew-formulae/apache-arrow.rb

```diff
@@ -15,7 +15,7 @@ class ApacheArrow < Formula
   depends_on "numpy"
   depends_on "openssl@1.1"
   depends_on "protobuf"
-  depends_on "python"
+  depends_on "python@2"
```

Comment:
That's a bit evil. Do we want to build ARROW_PYTHON at all in the Homebrew recipe?

Reply:
I was just trying it, but in hindsight I don't think it will work (PYTHON_EXECUTABLE looks for python3).

My next attempt is removing python2: there was this suggestion on the failed job that I missed before https://travis-ci.org/ursa-labs/crossbow/builds/631795636#L1793

If that fails, then we may have to turn off ARROW_PYTHON for now, unless someone has a better idea.